pypeit/scripts/chk_tilts.py

```diff
@@ -36,6 +36,7 @@ def main(pargs):
 
     import subprocess
 
+    # TODO: THIS IS OUT OF DATE
```

Comment:
I agree.

Reply:
Okay, I've added a note at the top of the script (which propagated to readthedocs), but I'm leaving the script in case someone eventually updates it.